extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -463,7 +463,15 @@ Adds support for Permutive event collection. Additionally, the following `vars`
 -   `namespace`: your Permutive AMP namespace
 -   `key`: your Permutive public API key
 
-Use `extraUrlParams` to add additional event properties. Full configuration details can be found at [support.permutive.com](http://support.permutive.com).
+Use `extraUrlParams` to add additional event properties. Full configuration details can be found at [support.permutive.com](https://support.permutive.com).
+
+### Permutive-ampscript
+
+Type attribute value: `permutive-ampscript`
+
+Adds support for Permutive event collection via the Permutive SDK running inside `amp-script`. This is a newer version of the `Permutive` configuration mentioned above. It will soon replace the old configuration.
```

Comment:
When will it replace the old one? I think we could give more clear direction.

- In the `# Permutive` section, write that it is legacy and new code should use `Permutive-ampscript`.
- In the `# Permutive-ampscript` section say that it is preferred vs. older `# Permutive`

Reply:
Its not quite so simply unfortunately. We are still beta testing the code that will receive the amp-analytics events. The reason I'm hesitant to mention that new code should be used is since we're onboarding new customers while the beta is ongoing, they will also be onboarded to the older config for now.

In that sense, mentioning that one is preferred might be misleading for now but it will make sense once beta ends. 

Would it be ok if we made this change after we move our code from beta to production?